infra/gcp/lib.sh

```diff
@@ -33,6 +33,9 @@ PROMOTER_SVCACCT="k8s-infra-gcr-promoter"
 # The service account name for the image promoter's vulnerability check.
 PROMOTER_VULN_SCANNING_SVCACCT="k8s-infra-gcr-vuln-scanning"
 
+# The service account name for the k8s vulnerability dashboard.
+VULN_DASHBOARD_SVCACCT = "k8s-infra-gcr-vuln-dashboard"
```

Comment:
agh, I should have caught this, PR up shortly
```suggestion
VULN_DASHBOARD_SVCACCT="k8s-infra-gcr-vuln-dashboard"
```

Reply:
Opened https://github.com/kubernetes/k8s.io/pull/1142